CMakeLists.txt

```diff
@@ -980,6 +980,10 @@ else()
   set(DOLPHIN_WC_IS_STABLE "0")
 endif()
 
+# Remove in-tree revision information generated by Visual Studio
+# This is because CMake will check in-tree first and use this, even if it is outdated
```

Comment:
It's not really CMake that checks, it's the compiler due to the given order of include paths.